lambdas/data-migration1/src/providers.ts

```diff
@@ -0,0 +1,157 @@
+import Knex from 'knex';
+import isNil from 'lodash/isNil';
+
+import DynamoDbSearchQueue from '@cumulus/aws-client/DynamoDbSearchQueue';
+import * as KMS from '@cumulus/aws-client/KMS';
+import { envUtils, keyPairProvider } from '@cumulus/common';
+import Logger from '@cumulus/logger';
+
+import { RecordAlreadyMigrated } from './errors';
+import { MigrationSummary } from './types';
+
+const Manager = require('@cumulus/api/models/base');
+const schemas = require('@cumulus/api/models/schemas');
+
+const logger = new Logger({ sender: '@cumulus/data-migration/providers' });
+
+export interface RDSProviderRecord {
```

Comment:
I guess it's fine that this lives here for now but, as soon as we write code that's reading or writing to the DB, we should move this somewhere more widely accessible.